@utahdts/utah-design-system/react/components/buttons/ConfirmationButton.jsx

```diff
@@ -42,7 +53,25 @@ const defaultProps = {
   size: formElementSizesEnum.MEDIUM,
   type: 'button',
 };
-
+/** @typedef {import('../../jsDocTypes').EventAction} EventAction */
+/**
+ * @param {Object} props
+ * @param {ButtonAppearance} [props.appearance]
```

Comment:
Should we have an appearance for the confirmation state? Right now the only way to change the second appearance is css? Is it weird that its a outline button --> solid button on confirm?

Reply:
There is a visual bug with a outline button when its transitioned to the confirmation state. If you hover your mouse off the button the text is the wrong color.

Reply:
add Prop: promptColor

or use

confirmationChildren
confirmationColor